docs/spec/v1beta1/event.md

```diff
@@ -58,9 +58,17 @@ package to push events to notification-controller API.
 
 ## Rate limiting
 
-Events sent to the notification-controller are subject to rate limiting to reduce the amount of duplicate alerts sent by notification-controller. Events are rate limited based on its `InvolvedObject.Name`, `InvolvedObject.Namespace`, `InvolvedObject.Kind`, and `Message` and if present in the metadata `revision`. The interval of the rate limit is set by default to `5m` but can be configured with the `--rate-limit-interval` option.
+Events received by notification-controller are subject to rate limiting to reduce the
+amount of duplicate alerts sent to external systems like Slack, Sentry, etc.
+
+Events are rate limited based on `InvolvedObject.Name`, `InvolvedObject.Namespace`,
+`InvolvedObject.Kind`, `Message` and `revision` if present in the metadata.
```

Comment:
```suggestion
`InvolvedObject.Kind`, `Message` and `Revision`, if present in the metadata.
```

Reply:
See https://github.com/fluxcd/notification-controller/blob/main/internal/server/event_server.go#L150

Reply:
```suggestion
`InvolvedObject.Kind`, `Message`, and `revision` if present in the metadata.
```

then, as at present the sentence reads as if all the fields are in the metadata.

Reply:
Fixed